### PR TITLE
Update to Gradle 5.6.1, Java 11 & gradle-plugin 1.6.11

### PR DESCRIPTION
**Pull request by JakubVanek**
This switches everything to Java 11 (source & target compatibility, travis JDK, docker JDK) and upgrades to Gradle 5.6.1 and gradle-plugin 1.6.9.

### CODE REVIEW

**Comment by JakubVanek** [comment]
The build will fail now, because the plugin version isn't published yet.

File: gradlew [hunk 1]
```diff
@@ -28,7 +44,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```
Comment (jabrena) [inline gradlew]: Is it necessary?

Comment (jabrena) [inline gradlew]: Parameters

File: gradlew.bat [hunk 2]
```diff
@@ -14,7 +30,7 @@ set APP_BASE_NAME=%~n0
 set APP_HOME=%DIRNAME%
 
 @rem Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-set DEFAULT_JVM_OPTS=
+set DEFAULT_JVM_OPTS="-Xmx64m" "-Xms64m"
```
Comment (jabrena) [inline gradlew.bat]: Parameters

**Review by jabrena — APPROVED** [review]

**Pull request merged**